test/simple_source/stmts/00_docstring.py

```diff
@@ -4,18 +4,42 @@
 # RUNNABLE!
 r'''func placeholder - with ("""\nstring\n""")'''
 
-def uni(word):
-  u"""        <----- SEE 'u' HERE
-  >>> mylen(u"áéíóú")
-  5
-  """
+def dq0():
+    assert __doc__ == r'''func placeholder - with ("""\nstring\n""")'''
 
+def dq1():
+    """assert that dedent() has no effect on 'text'"""
+    assert dq1.__doc__ == """assert that dedent() has no effect on 'text'"""
 
-def foo():
+def dq2():
+    '''assert that dedent() has no effect on 'text\''''
+    assert dq1.__doc__ == '''assert that dedent() has no effect on 'text\''''
+
+def dq3():
+    """assert that dedent() has no effect on 'text\""""
+    assert dq3.__doc__ == """assert that dedent() has no effect on 'text\""""
+
+def dq4():
+    """assert that dedent() has no effect on 'text'"""
+    assert dq4.__doc__ == """assert that dedent() has no effect on 'text'"""
+
+def dq5():
     r'''func placeholder - ' and with ("""\nstring\n""")'''
+    assert dq5.__doc__ == r'''func placeholder - ' and with ("""\nstring\n""")'''
 
-def bar():
+def dq6():
     r"""func placeholder - ' and with ('''\nstring\n''') and \"\"\"\nstring\n\"\"\" """
+    assert dq6.__doc__ == r"""func placeholder - ' and with ('''\nstring\n''') and \"\"\"\nstring\n\"\"\" """
+
+def dq7():
+  u"""        <----- SEE 'u' HERE
```

Comment:
These lines need more indentation same as others, otherwise test will fail.

Reply:
Just to be clear. If you run this from python, I believe it doesn't give an error. It is just after decompiling. And decomping 3.7 byecode  works using unpy37. 

There is some fancy indentation code in uncompyle6 that unpy37 doesn't have. Perhaps that should be removed?

Reply:
Ok i checked `unpy37`, we use passed arg `indent` in `print_docstring` to print line-by-line. However the content of multiline docstring treat the string as raw:
```python
def t():
  """
test
  """
  assert t.__doc__ == """
test
  """

t()
```

is decompiled to :

```python
def t():
    """
    test
      """
    print(t.__doc__)
    if not t.__doc__ == '\ntest\n  ':
        raise AssertionError

t()
```

I believe we should not [parse](https://github.com/rocky/python-uncompyle6/blob/master/uncompyle6/semantics/helper.py#L161) docstring and use docstring as is. then our indent is ok.

Reply:
This fixes.
```diff
@@ -184,12 +184,12 @@ def print_docstring(self, indent, docstring):
         self.println(trimmed[0])
         for line in trimmed[1:-1]:
             if line:
-                self.println( indent, line )
+                self.println( line )
             else:
                 self.println( "\n\n" )
                 pass
             pass
-        self.println(indent, trimmed[-1], quote)
+        self.println(trimmed[-1], quote)
```

Reply:
Got it. Thanks!

Reply:
This change has been committed although, dq7 (unicode accents) fails on 3.{6,7}

Reply:
@rocky, I'm a little confused about [these](https://github.com/rocky/python-uncompyle6/blob/master/uncompyle6/semantics/helper.py#L162-L172) lines. Why we need this?

Removing those lines and `trimmed = lines` (ugly) works in all versions as i checked. Could you please verify?